insights/combiners/rhel_for_edge.py

```diff
@@ -13,9 +13,11 @@
 from insights.parsers.installed_rpms import InstalledRpms
 from insights.parsers.systemd.unitfiles import ListUnits
 from insights.parsers.redhat_release import RedhatRelease
+from insights.parsers.rpm_ostree_status import RpmOstreeStatus
+from insights.parsers import SkipComponent
```

Comment:
This `SkipComponet` is moved to the `insights.core.exceptions` module, please import it from there.

Reply:
@xiangce - Updated.